worker/main.ts

```diff
@@ -0,0 +1,39 @@
+import { config } from 'dotenv';
+
+import { MoveContactJobWorker } from './jobs/move-contact/move-contact-job.worker';
+import { QUEUE_NAMES } from '../shared/queues';
+
+config();
+
+interface WorkerConfig {
+  queueName: string;
+  workerClass: new (queueName: string) => any;
+}
+
+// Define the configuration for workers
+const workerConfigs: WorkerConfig[] = [
+  {
+    queueName: QUEUE_NAMES.MOVE_CONTACT_QUEUE,
+    workerClass: MoveContactJobWorker
+  },
+  // Add new workers here
+];
+
+
+async function main() {
+  for (const { queueName, workerClass } of workerConfigs) {
+    try {
+      new workerClass(queueName);
+    } catch (error) {
+      console.error(`Error starting worker for queue "${queueName}":`, error);
+      throw error;
+    }
+  }
+
+  console.log(`🚀 All your workers are listening`);
+}
+
+main().catch((error) => {
```

Comment:
is catching this helpful? typically errors are automatically written to console and cause an error state, so this shouldn't be needed and might actually reduce what gets logged (eg. where is the stack?)